x-pack/plugin/old-lucene-versions/src/main/java/org/elasticsearch/xpack/lucene/bwc/codecs/lucene62/Lucene62Codec.java

```diff
@@ -54,6 +57,21 @@ public DocValuesFormat getDocValuesFormatForField(String field) {
             return defaultDocValuesFormat;
         }
     };
+    private final PostingsFormat postingsFormat = new LegacyAdaptingPerFieldPostingsFormat() {
+        @Override
+        public PostingsFormat getPostingsFormatForField(String field) {
```

Comment:
Can we move this check to the `LegacyAdaptingPerFieldPostingsFormat` class?

Reply:
👍